js/docs.js

```diff
@@ -197,10 +204,11 @@ var docs = {
     'ua': '<p>Визначає розмір і назви для grid рядків і стовпців.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
     'no': '<p>Spesifiserer størrelsene og navnene på radene og kolonnene til rutenettet.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
     'pt-pt': '<p>Especifica as dimensões e nomes das linhas e colunas da grelha.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
-    'bg': '<p>Определя размера и имената на редовете и колоните на решетката.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>'
+    'bg': '<p>Определя размера и имената на редовете и колоните на решетката.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
+    'he': '<p>קובע את הגודל והשמות של השורות והעמודות של הסריג.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
   },
   'grid-template-areas': {
-    'en': '<p>Specifies named grid areas.</p><p><code>&lt;grid-name&gt;</code></p>',
+    'en': '<p>Specifies named grid areas.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
```

Comment:
Can you remove this change from `<grid-name>` to `<grid-template-rows> / <grid-template-columns>`?

Reply:
should be done